docs/deployment/configuration.md

```diff
@@ -0,0 +1,93 @@
+# Configuration
+
+This section explain the current configuration options that are available for the different components that are deployed but the Helm charts.
```

Comment:
```suggestion
This section explains the configuration options that are available for the different components deployed by the InvenioRDM Helm chart.
```

(it's one chart for InvenioRDM, right?)

Reply:
not really, is just the image that is configurable. Once we set it up we will put it in the docs, but this charts suit all Invenio instances.

Reply:
To be clear, in the updated configuration.md, what do "these Helm charts" refer to? Is the invenio chart an amalgamation of charts (or maybe what I am saying makes no sense in Kubernetes world?)